app/src/main/java/ch/epfl/sdp/map/MapFragment.java

```diff
@@ -258,6 +272,14 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
         mapView.onSaveInstanceState(outState);
     }
 
+    @Override
+    public void onClick(View view) {
+        System.out.println(view.getId());
```

Comment:
Should be removed I think

Reply:
Mmh not sure, I think that's from Adrien? (was there before)

Reply:
it prints something in the log, that's for debugging